docs/source/development/reference/functions.rst

```diff
@@ -828,8 +828,30 @@ Various
 
     Convert radians to string representation of degrees, minutes and seconds.
 
+    Note: starting with PROJ 9.2, this function is deprecated by
+    :cpp:func:`proj_rtodms2`
```

Comment:
Shouldn't the [deprecated directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated) be used used? (does it work?)
```suggestion
    .. deprecated:: 9.2
       Use :cpp:func:`proj_rtodms2` instead.
```